xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3104,7 +3104,6 @@ void CVideoPlayer::Pause()
   if (GetSpeed() == 0)
   {
     SetSpeed(1);
```

Comment:
what exactly do you consider dead code?

please move both callbacks to SetSpeed